Emby.Server.Implementations/Udp/UdpServer.cs

```diff
@@ -40,8 +40,7 @@ public UdpServer(ILogger logger, IServerApplicationHost appHost, IJsonSerializer
             _json = json;
             _socketFactory = socketFactory;
 
-            AddMessageResponder("who is EmbyServer?", true, RespondToV2Message);
-            AddMessageResponder("who is MediaBrowserServer_v2?", false, RespondToV2Message);
```

Comment:
Will removing these break the Emby clients?

Reply:
At this point Emby clients are working with Jellyfin Server... (Roku tested on my setup) It MAY be worth keeping the Emby response and adding Jellyfin  response until such time as Emby Clients stop working. (Manual Connect will still work without this)

Reply:
> Additionally `MediaBrowserServer_v2` seems like a generic _service_ name rather than a specific implementation of one and so might be worth keeping even longer. Like _Kleenex_ is to _facial tissue_.

Emby used to be called "MediaBrowser"

>Will removing these break the Emby clients?

There isn't really any point trying to be compatible, as they will eventually break it anyway with app updates. We will have our own apps soon enough. And you can manually enter server IP addresses to apps which is the alternative to them discovering servers

Reply:
You can also test this now by checking out this branch and building it so we know if that's the actual case.

Reply:
I think it won't hurt to keep backward compatibility a little longer.
Also remember that we are not the ones who were trying to break this compatibility ;)